src/pudl/output/ferc1.py

```diff
@@ -1477,6 +1485,22 @@ def generate_intertable_calculations(
             table_name=self.root_table,
             add_corrections=True,
         )
+        logger.info("Checking sub-total calcs.")
+        subtotal_calcs = pudl.transform.ferc1.calculate_values_from_components(
+            calculation_components=calculations_intertable[
+                calculations_intertable.is_total_to_subdimensions_calc
+            ],
+            data=exploded,
+            calc_idx=calc_idx,
+            value_col=self.value_col,
+        )
+        subtotal_calcs = pudl.transform.ferc1.check_calculation_metrics(
+            calculated_df=subtotal_calcs,
+            value_col=self.value_col,
+            calculation_tolerance=self.calculation_tolerance.intertable_calculation_errors,
+            table_name=self.root_table,
+            add_corrections=True,
+        )
```

Comment:
there are no total to sub-dimension inter-table calcs rn but this felt like a good thing to run nonetheless